src/components/TransactionItemComponent/DataCeils/TypeCell.tsx

```diff
@@ -0,0 +1,53 @@
+import useTheme from '@hooks/useTheme';
+import Icon from '@components/Icon';
+import CONST from '@src/CONST';
+import React from 'react';
+import TextWithTooltip from '@components/TextWithTooltip';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as Expensicons from '@components/Icon/Expensicons';
+import type DataCellProps from './DateCellProps';
+
+const getTypeIcon = (type?: string) => {
+    switch (type) {
+        case CONST.EXPENSE.TYPE.CASH_CARD_NAME:
+            return Expensicons.Cash;
+        case undefined:
+            return Expensicons.Cash;
+        default:
+            return Expensicons.Cash;
```

Comment:
It always returns `Expensicons.Cash` 🤔

Reply:
in last case it should be card, good catch